content/_index.md

```diff
@@ -14,21 +14,21 @@ carousel:
     link: https://www.cs.cmu.edu/news/2023/delphi-cdc-center
     alt: Learn More
     image: landing-pg-hero-_cfa_oadm.jpg
-  - pre: Tools
-    title: Evaluation of COVID-19 Forecasts
-    link: /forecast-eval
-    alt: Explore the Forecast Evaluation Dashboard
-    image: forecast_evaluation_dashboard.jpg
-  - pre: Tools
-    title: COVID Trends and Impact Survey (CTIS)
-    ref: covidcast/survey-results
-    alt: Explore the CTIS Dashboard
-    image: covidcast_survey.jpg
+  - pre: Blog
+    title: Exploratory Analysis - Identifying Changing Variant Behavior during a Pandemic
+    ref: 2023-12-20-changepoint_explore
+    alt: View Post
+    image: landing-pg-hero-exploratory_analysis.jpg
   - pre: Blog
     title: On the Predictability of COVID-19
     ref: 2021-09-30-ensemble-analysis
     alt: View Post
     image: ensemble-hero.jpg
+  - pre: Tools
+    title: An Open Repository of Real-Time COVID-19 Indicators
+    link: https://delphi.cmu.edu/blog/2022/12/14/introducing-epidata-v4/
+    alt: Introducing Epidata
```

Comment:
This title isnt appropriate because the article is not at all COVID specific...  And then do we want to move this one slot closer to the beginning of the carousel (in front of the COVID-titled article)?
```suggestion
    title: Introducing Epidata
    link: https://delphi.cmu.edu/blog/2022/12/14/introducing-epidata-v4/
    alt: View Post
```

Reply:
Good catch! I do think that moving it closer to the beginning is a good idea, too.